cmd/kube-proxy/app/server.go

```diff
@@ -104,11 +112,15 @@ type Options struct {
 	master string
 	// healthzPort is the port to be used by the healthz server.
 	healthzPort int32
+
+	scheme *runtime.Scheme
```

Comment:
I wanted to share the scheme with the `applyDefaults` function, and hopefully also where we set up the event recorder (since it needs a scheme). Is there a better way?